pages/pipelines/create_your_own.md

```diff
@@ -77,7 +77,7 @@ To create a new pipeline:
     ```
 
 1. Select _Create Pipeline_.
-1. On the next page showing your pipeline name, click _New Build_ and in the modal that opens, create a build using the pre-filled details.
+1. On the next page showing your pipeline name, click _New Build_. In the modal that opens, create a build using the pre-filled details.
```

Comment:
Lol looks like I made the same change locally but didn't commit it. Anyway, great mind thing alike. I added your suggested commit.